### DISCUSSION TITLE
Unable to buy jfree chart developper guide

### DISCUSSION THREAD

**Discussion by JohanThomass** [jfree/jfreechart#308] [General]
Hi,

I want to buy the jfree chart developper guide but I got this message "Payments can't be processed right now because of a technical issue. Try again in a few minutes" on the website https://efinery.myshopify.com

Maybe this website is not maintained any more ?

How could I get the latest guide please ?

Thanks,

Best regards

Johan

**Comment by trashgod** [comment]
See also issues #302 and #253.
